js/src/edit-block/components/keywords-section.js

```diff
@@ -32,6 +33,22 @@ const KeywordsSection = () => {
 		changeBlock( { keywords: tokens } );
 	};
 
+	const getKeywords = useCallback(
+		/**
+		 * Gets the block keywords.
+		 *
+		 * @return {Array} The block keywords, if any.
+		 */
+		() => {
+			if ( ! Array.isArray( block.keywords ) ) {
+				return [];
+			}
+
+			return block.keywords.filter( ( keyword ) => Boolean( keyword ) );
```

Comment:
The previous editor sometimes saved keywords as `[ '' ]`, which looked like:

![keyword-hisd](https://user-images.githubusercontent.com/4063887/102300283-425ca700-3f1a-11eb-9abe-609ed86de149.png)

No need to display a `''` keyword.